configuration/scripts/options/set_nml.jra55_gx3

```diff
@@ -0,0 +1,17 @@
+year_init      = 2005
+ice_ic  = 'ICE_MACHINE_INPUTDATA/CICE_data/ic/gx3/iced_gx3_v5.nc'
+grid_file   = 'ICE_MACHINE_INPUTDATA/CICE_data/grid/gx3/grid_gx3.bin'
+kmt_file    = 'ICE_MACHINE_INPUTDATA/CICE_data/grid/gx3/kmt_gx3.bin'
+bathymetry_file = 'ICE_MACHINE_INPUTDATA/CICE_data/grid/gx3/global_gx3.bathy.nc'
+use_leap_years = .true.
+use_restart_time = .false.
+maskhalo_dyn   = .true.
+maskhalo_remap = .true.
+maskhalo_bound = .true.
+fyear_init      = 2005
+atm_data_dir    = '/p/work1/allard/CICE_data/forcing/gx3/JRA55'
```

Comment:
Same.  There seem to be others I won't point out.

Reply:
I believe I have fixed all the files so they all should point to "ICE_MACHINE_INPUTDATA"